tests/other_test.go

```diff
@@ -0,0 +1,24 @@
+// Tests in this file are NOT run in the PR pipeline. They are run in the continuous testing pipeline along with the ones in pr_test.go
+package test
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+	"testing"
+)
+
```

Comment:
perhaps leave a comment saying this test will fail until SCC backend issue is fixed -> https://github.ibm.com/project-fortress/pm/issues/14661